packages/test-runner/src/nodeResolvePlugin.ts

```diff
@@ -10,6 +10,10 @@ export function nodeResolvePlugin(
   const options: RollupNodeResolveOptions = {
     rootDir,
     extensions: ['.ts', '.tsx', '.jsx', '.mjs', '.js', '.json'],
+    customResolveOptions: {
+      moduleDirectory: ['node_modules', 'web_modules'],
```

Comment:
we also need to merge in `userOptionsObject. customResolveOptions` and `userOptionsObject. customResolveOptions. moduleDirectory `. Or use deepmerge :)